pkg/image/filefmt.go

```diff
@@ -67,10 +67,12 @@ func CopyKnownHdrs() Headers {
 	return m
 }
 
+// Match performs a check to see if the provided byte array matches the bytes in our header data
```

Comment:
nit: technically byte slice not array...

Reply:
done